storybook/stories/API/polar/PolarGrid.stories.tsx

```diff
@@ -7,19 +7,15 @@ export default {
 
 export const Simple = {
   render: (args: Record<string, any>) => {
-    const { polarAngles, polarRadius } = args;
     return (
       <Surface width={500} height={500}>
         <PolarGrid
           cx={250}
           cy={250}
           innerRadius={0}
           outerRadius={200}
-          width={500}
-          height={500}
-          polarAngles={polarAngles}
-          polarRadius={polarRadius}
-          radialLines
+          polarAngles={args.polarAngles}
+          polarRadius={args.polarRadius}
```

Comment:
As the controls have all properties of the polar grid, I think this would be best.
```suggestion
          {...args}
```